### DISCUSSION TITLE
tpu acceleration

### DISCUSSION THREAD

**Discussion by Salupol** [voicepaw/so-vits-svc-fork#536] [Q&A] [answered]
good day, my ai lovers
I will have coral usb tpu accelerator on this week and I want to use it to make new model.
is this supported by default and I only to install runtime (or something more?)to make this work or I need to do something else? 
thanks for ur answers <3

**Comment by 34j** [comment, marked as answer]
We gave up on TPU support because it was too difficult. Anyone can send a PR.
